<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>832</width>
    <height>632</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="mainWidget" native="true">
    <property name="geometry">
     <rect>
      <x>18</x>
      <y>18</y>
      <width>800</width>
      <height>600</height>
     </rect>
    </property>
    <widget class="titlebar" name="titleBar" native="true">
     <property name="geometry">
      <rect>
       <x>18</x>
       <y>18</y>
       <width>800</width>
       <height>30</height>
      </rect>
     </property>
    </widget>
    <widget class="sidemenu" name="sideMenu" native="true">
     <property name="geometry">
      <rect>
       <x>18</x>
       <y>47</y>
       <width>100</width>
       <height>570</height>
      </rect>
     </property>
    </widget>
    <widget class="QWidget" name="displayArea" native="true">
     <property name="geometry">
      <rect>
       <x>118</x>
       <y>48</y>
       <width>700</width>
       <height>570</height>
      </rect>
     </property>
    </widget>
    <zorder>sideMenu</zorder>
    <zorder>titleBar</zorder>
    <zorder>displayArea</zorder>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>sidemenu</class>
   <extends>QWidget</extends>
   <header>sidemenu.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>titlebar</class>
   <extends>QWidget</extends>
   <header location="global">titlebar.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
